lib/debt_to_income_ratio_latest/main.py

```diff
@@ -134,12 +139,15 @@ async def get_debt_to_income_ratio_latest(
     client = AsyncClient(token)
 
     now = datetime(2021, 9, 1)
+    cutoff_interval = timedelta(days=30)
```

Comment:
Still need to make a case to return None if sum_of_depository_credit_transactions = 0 on line 130?

Reply:
That has been addressed at #63 !